docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,1918 @@
+{
```

Comment:
<p>I would maybe change a little to have a broader introduction (it's mentioned below but it's good to reiterate):</p><blockquote>The <code>.X</code> slot holds the matrix of gene expression data that we are actively using for analysis (e.g. for plotting gene expression values). In this example, <code>adata.X</code> holds the matrix of normalized and [...]</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Good idea, added as suggested! </p>